thirdparty/build_thirdparty.sh

```diff
@@ -74,6 +75,13 @@ if [ -n "$F_ALL" -o -n "$F_LZ4" ]; then
   make -j$PARALLEL install
 fi
 
+# build zlib
+if [ -n "$F_ALL" -o -n "$F_ZLIB" ]; then
+  cd $TP_DIR/$ZLIB_BASEDIR/cmake_unofficial
+  CFLAGS=-fPIC cmake -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX $ZLIB_DIR
+  make -j$PARALLEL install
```

Comment:
It looks like zlib does have a CMakeLists.txt, but it's in the base directory of the tarball, so remove the `/cmake_unofficial`